File: KuantumKaosYonetimi/Javascript/KuantumKaosYonetimi/IKritik.js
// IKritik.js

/**
 * B. Arayüz (Interface Segregation)
 * Kritik nesnelerin uygulayacağı sözleşme.
 */
class IKritik {
    AcilDurumSogutmasi() {
        // Bu, alt sınıfların implemente etmesi gereken bir metottur.
        throw new Error("AcilDurumSogutmasi() metodu IKritik nesneler tarafından uygulanmalıdır.");
    }
}

module.exports = IKritik;

File: KuantumKaosYonetimi/Python/KuantumKaosYonetimi/i_kritik.py
# i_kritik.py
from abc import ABC, abstractmethod

# B. Arayüz (Interface Segregation karşılığı)
class IKritik(ABC):
    
    @abstractmethod
    def AcilDurumSogutmasi(self):
        """Bu metot çağrıldığında nesnenin stabilitesi +50 artar (Max 100)."""
        pass